src/opera/util/metadata_utils.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python3
+#
+
+"""
+============
+metadata_utils.py
+============
+
+ISO metadata utilities for use with OPERA PGEs.
+
+"""
+
+from osgeo import gdal
+from osgeo import osr
+import mgrs
+
+def get_geographic_boundaries_from_mgrs_tile(mgrs_tile_name, verbose=False):
```

Comment:
A nice to have for this function would be if it could check `mgrs_tile_name` to see if it starts with a "T", and if so, remove it. The tile codes we parse from input HLS datasets usually start with a "T", but `mgrs.MGRSToUTM` does not expect it, so this function is the best place to do the input sanitation.